### ISSUE TITLE
Miracle solve makes me suspicious...

### ISSUE THREAD

**Issue by mdda** [samacqua/LARC#3]
In the following problem, "Description 3" is plainly insufficient, however "Builder 3" is able to solve it directly, despite having practically none of the required information (eg: how did they know even basic things like the grid size, or colour to use?):

https://samacquaviva.com/LARC/explore/description.html?task=398&id=e6f40d2d-9aed-4725-904d-15ccbceaa704

What's going on?

**Comment by samacqua** [comment]
Yes, there are a small number of these. I think there are 3 possible explanations:
1. The worker used the javascript code to cheat. The checking for if a solution is correct was done locally. In theory, the worker could have looked at the javascript, loaded the input-output examples themselves, and used that to submit the right answer.
2. The worker found the ARC dataset online and found this particular task in the dataset.
3. The worker had done the task before with a different account and had already seen this task.

I think #1 is the most likely because the code was not obfuscated or anything.
